README.md

```diff
@@ -13,3 +13,17 @@ updating YAML files in a git repository, and committing the changes.
 
 Please see the [installation and use
 guide](https://toolkit.fluxcd.io/guides/image-update/).
+
+## How to work on it
+
+You will need to install `libgit2` corresponding to the version of its
```

Comment:
Wrote detailed instructions for this for [source-controller](https://github.com/fluxcd/source-controller/blob/main/CONTRIBUTING.md#installing-required-dependencies) from which you may want to piggyback.